src/ynEIGEN/AssetRegistry.sol

```diff
@@ -286,8 +286,10 @@ interface IAssetRegistryEvents {
             revert LengthMismatch(assetsCount, stakedAssetBalances.length);
         }
 
+        IRedemptionAssetsVaultExt redemptionAssetsVault = strategyManager.redemptionAssetsVault();
         for (uint256 i = 0; i < assetsCount; i++) {
             assetBalances[i] += stakedAssetBalances[i];
+            if (address(redemptionAssetsVault) != address(0)) assetBalances[i] += redemptionAssetsVault.balances(address(assets[i]));
```

Comment:
can we implement a function here that does:
redemptionAssetsVault.assetBalances() that works like this:

https://github.com/yieldnest/yieldnest-protocol/blob/753bec9eaef6ee09310bfb9826b71d841e2721fd/src/ynEIGEN/ynEigen.sol#L265

to have just 1 contract all?

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/164/commits/d78bbc0dda27352e0e1de19231fd01880b9c7bba